lib/qwerty/quran_simple.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'text'
+require_relative 'config'
+module Qwerty
+  class Text
+    class QuranSimple < Ruote::Participant
+      def on_workitem
+          workitem.fields['text'][:quran_simple] = {
+              :source => "http://tanzil.info",
+              :languageList => Qwerty.configuration.language_list
+            } 
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.<br>Trailing whitespace detected.